CONTRIBUTORS.md

```diff
@@ -0,0 +1,22 @@
+Contributors
+============
+
+* [Ajay Bhargav](https://github.com/ajaybhargav)
```

Comment:
Thanks, I will crosscheck this again and add missing names. As for the copyright: I'm not too familiar with how this works, but I was thinking about changing it to:
C 2009-2016 Dave Gamble and cJSON contributors